variables.tf

```diff
@@ -229,3 +198,60 @@ variable "manager_pod_annotations" {
   default     = {}
 }
 
+variable "additional_secrets" {
+  description = "additional secrets to mount into the manager pods"
+  type        = list(map(string))
+  default     = []
+}
+
+variable "replicas" {
+  description = "the number of manager pods to create"
+  type        = number
+  default     = 1
+}
+
+variable "runner_name" {
+  description = "name of the runner"
+  type        = string
+}
+
+variable "unregister_runners" {
+  description = "whether runners should be unregistered when pool is deprovisioned"
+  type        = bool
+  default     = true
+}
+
+variable "runner_token" {
+  description = "token of already registered runer. to use this var.runner_registration_token must be set to null"
+  type        = string
+  default     = null
+}
+
+
+variable "cache" {
+  description = "value"
+  type = object({
+    type   = string
+    path   = string
+    shared = bool
+    gcs    = map(any)
+    s3     = map(any)
+    azure  = map(any)
+  })
+
+  validation {
+    condition     = var.cache.type == "gcs" ? lookup(var.cache.gcs, "CredentialsFile", "") != "" || lookup(var.cache.gcs, "AccessID", "") != "" : true
+    error_message = "To use the gcs cache type you must set either CredentialsFile or AccessID and PrivateKey in var.cache.gcs."
+  }
+
+  default = {
+    type   = ""
+    shared = false
+    path   = ""
+    gcs = {
+      CredentialsFile = ""
```

Comment:
Why are we setting this?  the default too should be an empty map.

Reply:
I forgot about that. All corrected now